src/asciidoc/docs/develop/editor.adoc

```diff
@@ -78,3 +80,51 @@ You can scan this code to preview the template on your mobile device.
 === Launching the debugger
 
 *Launch Debugger* opens the debugger in a new browser tab.
+
+
+[.faq]
+== Tips
+[id="faq-nodejsnpmver"]
+.What versions of Node.js and npm do Kojis run on?
+[%collapsible]
+====
+include::{includespath}/misc.adoc[tag=nodejsnpmver]
+====
+
+[id="faq-revert-kojiversion"]
+.How do I revert to an old version of my template?
+[%collapsible]
+====
+To revert to a previous commit,
+
+include::{includespath}/steps-codeeditor.adoc[tag=openproject-template2edit]
+
+include::{includespath}/steps-codeeditor.adoc[tag=expand-terminal]
+
+. Run `git log` and copy the hash code of the commit you want to restore.
+
+. In the terminal panel, enter `git reset --hard <commit-hash>`.
++
+WARNING: You will lose all the changes you made after the commit you restore.
+
+TIP: Publish more frequently for a more detailed commit history.
+====
+
+[id="faq-reopen-frontend-terminal"]
+.How do I reopen the frontend terminal?
+[%collapsible]
+====
+To re-open the frontend terminal:
+
+. Click on the [.tabname]`New` tab.
+
+. Run:
++
+....
+cd frontend
+npm start
+....
+====
+
+
+// DC-483 DC-485
```

Comment:
Remove this comment. We aren't going to include internal ticket tracking in the source code for every page. If we need it, we can see the ticket numbers in the commit history of the repo.

Reply:
Fixed. Will be in the next commit.